Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -359,7 +359,7 @@ public enum FeatureFlag: String, CaseIterable {
         case .useDescriptiveActionAttributedTextView:
             true
         case .newOnboardingUpgrade:
-            false
+            true
```

Comment:
Is it ok to enable this here or was this just for testing?

Reply:
Just for testing!